app/src/main/res/layout/item_search_result.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I didn't want to make it explicitly about speaker because I think we may reuse this layout for the topics in the same page. So maybe `item_search_generic`?